templates/main.Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM scratch
+
+COPY manifests /manifests
+COPY metadata/annotations.yaml /metadata/annotations.yaml
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=__NAME__
+LABEL operators.operatorframework.io.bundle.channel.default.v1="__DEFAULT_CHANNEL__"
+LABEL operators.operatorframework.io.bundle.channels.v1="__CHANNEL_LIST__"
+
+LABEL \
+      com.redhat.component="openshift-serverless-1-serverless-rhel8-operator-container" \
+      name="openshift-serverless-1/serverless-rhel8-operator" \
+      version="__VERSION__" \
+      summary="Red Hat OpenShift Serverless 1 Serverless Operator" \
+      maintainer="serverless-support@redhat.com" \
+      description="Red Hat OpenShift Serverless 1 Serverless Operator" \
+      io.k8s.display-name="Red Hat OpenShift Serverless 1 Serverless Operator" \
+      com.redhat.openshift.versions="v4.5" \
```

Comment:
This "v4.5" could be in the template as well.

Reply:
Yeah. But, I didn't know what it is. How to name this?! @warrenvw ?

Reply:
@cardil AFAIK, this means the target OCP version we're releasing for. This should also ensure "stage" index images are built for both 4.5 and 4.6. (when the other Warren's PR is merged)

Reply:
Done